doc/nrf/ug_multi_image.rst

```diff
@@ -129,58 +132,88 @@ For example:
 
-You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
+Add these configuration options to the Kconfig file of your child image, replacing ``<IMAGE_NAME>`` with the uppercase name of your child image, as specified in ``add_child_image``.
 
-This can be done by including the :file:`Kconfig.template.build_strategy` template, as shown below.
+This can be done by including the :file:`Kconfig.template.build_strategy` template as follows:
 
 .. code-block:: Kconfig
 
    module=MCUBOOT
    source "${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/Kconfig.template.build_strategy"
 
+.. _ug_multi_image_variables:
 
 Image-specific variables
 ========================
 
 The child image and parent image are executed in different CMake processes and thus have different namespaces.
 Variables in the parent image are not propagated to the child image, with the following exceptions:
 
-* Any variable named *IMAGENAME*\_FOO in a parent image is propagated to the child image named *IMAGENAME* as FOO.
-* Variables that are in the list ``SHARED_MULTI_IMAGE_VARIABLES`` are propagated to all child images.
+* Any variable named ``<IMAGE_NAME>\_FOO`` in a parent image is propagated to the child image named ``<IMAGE_NAME>`` as ``FOO``.
+* Variables that are in the ``SHARED_MULTI_IMAGE_VARIABLES`` list are propagated to all child images.
 
 With these two mechanisms, it is possible to set variables in child images from either parent images or the command line, and it is possible to set variables globally across all images.
-For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows::
+For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows:
+
+.. code-block::
 
    cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
 
 You can extend the CMake command that is used to create the child images by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
 For example, add ``--trace-expand`` to that variable to output more debug information.
 
+With ``west``, these configuration variables are passed into CMake by using the ``--`` separator:
+
+.. code-block::
+
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
+
+Any configuration option for the application or child image can be passed in this way.
+For example, use the following command to build the ``hello_world`` sample using multi-stage bootloaders and configurations specific to each bootloader image:
+
+.. code-block::
+
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -DCONFIG_SECURE_BOOT=y -DCONFIG_BOOTLOADER_MCUBOOT=y -DCONFIG_BUILD_S1_VARIANT=y -Db0_OVERLAY_CONFIG=overlay-minimal-size.conf -Dmcuboot_CONFIG_LOG_MINIMAL=y
```

Comment:
This becomes a very long rendered line.

Not sure that is really helpful.

Reply:
This is changed in #3607 to be more readable. A lot of the docs-specific changes were not considered here due to them changing again in that PR, as mentioned [here](https://github.com/nrfconnect/sdk-nrf/pull/3518#pullrequestreview-559201567).